example/lib/custom_code_box.dart

```diff
@@ -29,18 +29,14 @@ class _CustomCodeBoxState extends State<CustomCodeBox> {
   List<String?> get languageList {
     const TOP = <String>{
       "java",
-      "cpp",
       "python",
-      "javascript",
-      "cs",
-      "dart",
-      "haskell",
-      "ruby",
+      "scala",
+      "go",
     };
     return <String?>[
       ...TOP,
-      null, // Divider
-      ...CODE_SNIPPETS.keys.where((el) => !TOP.contains(el))
+      // null, // Divider
+      // ...CODE_SNIPPETS.keys.where((el) => !TOP.contains(el))
```

Comment:
The same as in PR #3. Try not to publish the same changes to different branches

Reply:
changes are copied as the PRs depend on each other